manager/testdata/new-blueprints/scenario3/plotter.yaml

```diff
@@ -136,10 +118,22 @@ spec:
       kind: M4DModule
       confChart:
         name: ghcr.io/mesh-for-data/arrow-flight-config-module:0.1.0
+      capabilities:
+      - capability: read
+        scope: cluster
```

Comment:
scope: workload

Reply:
I copied it from arrow-flight-config-module.yaml. I will change it there also. Thanks

Reply:
@revit13 Scenario 3 describes a multi-tenant flight server.  The scope should be clustser

Reply:
@simanadler Scenario 3 describes a config module for a multi-tenant server. Its scope is workload as it configures the server per workload. We do not deploy the server itself in the plotter/blueprint so the cluster scope is irrelevant.

Reply:
After re-thinking about that I agree with @shlomitk1 that the scope of a config-module should be a workload scope. This is because the config contains information such as credentials of the data assets accessed in the workload and it is set per workload. another question is what happens in multi-cluster env where the flight service is deployed in one cluster while it is read from applications from different clusters? the config module is applied per cluster?